.changeset/odd-panthers-ring.md

```diff
@@ -0,0 +1,5 @@
+---
+'@leafygreen-ui/banner': patch
+---
+
+Banner now forwards refs
```

Comment:
Technically a major change https://legacy.reactjs.org/docs/forwarding-refs.html#note-for-component-library-maintainers

Reply:
Ah interesting. Are we cool with releasing a major for Banner alongside this?

Reply:
Yeah, that's fine